src/style.css

```diff
@@ -1,3 +1,67 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  text-transform: uppercase;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.logo {
+  margin-left: 50px;
+  height: 40px;
+  text-decoration: none;
+  display: block;
+}
+
+.nav {
+  margin-right: 50px;
+}
+
+.nav__list {
+  margin: 0;
+  padding: 0;
+  display: flex;
+  list-style: none;
+}
+
+.nav__item {
+  margin-right: 20px;
+}
+
+.nav__link {
+  color: #000;
+  align-items: center;
```

Comment:
The 'align-items: center;' property in the '.nav__link' class is unnecessary because 'display: flex;' is not applied to any parent element that would require child alignment. Consider removing it to clean up the CSS.